packages/logrotate/extra/dcos-logrotate-agent.service

```diff
@@ -0,0 +1,7 @@
+[Unit]
+Description=Logrotate Mesos Master: Rotate Mesos agent logs
+[Service]
+Type=simple
+EnvironmentFile=/opt/mesosphere/environment
+ExecStartPre=/usr/bin/mkdir -p /var/log/mesos/archive
```

Comment:
Should this mkdir `/var/lib/dcos/mesos/log/archive` like the Master version does? Why the difference?